.buildkite/scripts/steps/scalability/benchmarking.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+#.buildkite/scripts/bootstrap.sh
+echo "--- yarn kbn reset && yarn kbn bootstrap"
+yarn kbn reset && yarn kbn bootstrap
+
+GCS_BUCKET="gs://kibana-performance/scalability-tests"
+GCS_ARTIFACTS_REL="gcs_artifacts"
+GCS_ARTIFACTS_DIR="${WORKSPACE}/${GCS_ARTIFACTS_REL}"
+KIBANA_LOAD_TESTING_DIR="${KIBANA_DIR}/kibana-load-testing"
+
+# These tests are running on static workers so we must delete previous build, load runner and scalability artifacts
+rm -rf "${KIBANA_BUILD_LOCATION}"
+rm -rf "${KIBANA_LOAD_TESTING_DIR}"
+rm -rf "${GCS_ARTIFACTS_DIR}"
+
+echo "--- Download the latest artifacts from single user performance pipeline"
+download_artifacts
+
+echo "--- Clone kibana-load-testing repo and compile project"
+checkout_and_compile_load_runner
+
+echo "--- Run Scalability Tests with Elasticsearch started only once and Kibana restart before each journey"
+cd "$KIBANA_DIR"
+node scripts/es snapshot&
+
+esPid=$!
+
```

Comment:
Instead of the direct `kill "$esPid"` below, we should probably use a [trap](https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_12_02.html)  to make sure Elasticsearch is stopped even when the script exits prematurely (e.g. build aborted due to timeout). This avoid errors in later builds. The following should do the trick:

```bash
trap "kill -9 $esPid" EXIT
```

Reply:
wow, that's great!